src/lib/alerts/generate-alert.ts

```diff
@@ -20,9 +20,9 @@ export const generateAlert = (options?: AlertOptions): Alert => {
   const longMessage = `${singleEquipment} ${adverb} ${errorType.toLowerCase()} at ${hhmmss}`;
 
   return {
-    id: faker.datatype.uuid(),
-    refId: options?.refId || '',
-    category: shuffle(dataOption.categories),
+    id: faker.string.uuid(),
+    contactRefId: options?.contactRefId || '',
+    category: shuffle(dataOption.categories) as Category,
```

Comment:
If we update the the categories property in the options.ts on line 4 to be of type Category[] we could then use the generic on shuffle<Category> instead of having to cast it.

Reply:
Great catch